po/de.po

```diff
@@ -54,63 +115,133 @@ msgstr ""
 
 #: src/exm-application.c:117
 msgid "Project Homepage"
-msgstr "Webseite"
+msgstr "Projekt Webseite"
+
+#: src/exm-browse-page.blp:7
+msgid "Search for extensions"
+msgstr "Suchen Sie nach Erweiterungen"
+
+#: src/exm-browse-page.blp:8
+msgid ""
+"Enter a keyword to search 'extensions.gnome.org' for GNOME Shell Extensions."
+msgstr ""
+"Geben Sie ein Keyword ein, um auf 'extensions.gnome.org' nach GNOME Shell "
```

Comment:
Keyword -> Suchbegriff?

Reply:
That was the word is was looking for thanks.